### PR TITLE
qa: remove libcephfs-java tests

### PR DESCRIPTION
**Pull request by batrick**
These have bit-rotted and no longer work. No cycles from interested parties
available to fix.

Fixes: https://tracker.ceph.com/issues/38487
Signed-off-by: Patrick Donnelly <pdonnell@redhat.com>

### CODE REVIEW

File: qa/packages/packages.yaml [hunk 1]
```diff
@@ -11,7 +11,6 @@ ceph:
   - python-ceph
   - libcephfs2
   - libcephfs-dev
-  - libcephfs-java
   - libcephfs-jni
```
Comment (gregsfortytwo) [inline qa/packages/packages.yaml]: Presumably this should go away too then.

File: qa/packages/packages.yaml [hunk 2]
```diff
@@ -43,7 +42,6 @@ ceph:
   - ceph-mgr-rook
   - ceph-mgr-ssh
   - ceph-fuse
-  - cephfs-java
   - libcephfs_jni1
```
Comment (gregsfortytwo) [inline qa/packages/packages.yaml]: And this.

File: qa/packages/packages.yaml (hunk 1 above)
Comment (batrick) [reply qa/packages/packages.yaml]: done

File: qa/packages/packages.yaml (hunk 2 above)
Comment (batrick) [reply qa/packages/packages.yaml]: done

**Comment by batrick** [comment]
jenkins render docs

**Comment by batrick** [comment]
>  We should probably note the untested state in the docs as well?

Sure, done!

> And do we still think it's worth keeping the code around, or should we just drop it entirely?

If the code still builds then I think it's okay to keep around. If it starts failing to build and isn't trivial to fix, then let's drop it.

**Comment by batrick** [comment]
http://docs.ceph.com/ceph-prs/26659/api/libcephfs-java/

LGTM. @gregsfortytwo ?

**Pull request merged**